src/mlpack/methods/ann/activation_functions/mish_function.hpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file mish_function.hpp
+ * @author Kartik Dutt
+ *
+ * Definition and implementation of the mish function as described by
```

Comment:
Should this be `Mish` function instead of `mish`?

Reply:
Hi zoq, I have made the required changes.
Thanks.